src/main/kotlin/com/approvers/devlazaApi/model/Projects.kt

```diff
@@ -42,26 +46,30 @@ data class SitesPoster(
 )
 
 @Entity
+@Table(indexes = [Index(name="tags_index", columnList="NAME", unique=true)])
 data class Tags(
         @Id @GeneratedValue(generator = "uuid2") @GenericGenerator(name = "uuid2", strategy = "uuid2") @Column(columnDefinition = "BINARY(16)") var id: UUID? = null,
         @Column(name="name", nullable = false) var name: String
```

Comment:
```suggestion
        @Column(name="name", nullable = false, unique=true) var name: String
```
uniqueならunique縛りしてください

Reply:
😇